security/selinux.ksy

```diff
@@ -0,0 +1,1605 @@
+meta:
+  id: selinux
+  title: SELinux file policy binary
+  license: CC0-1.0
+  encoding: ASCII
+  file-extension: bin
+  endian: le
+doc-ref: |
+  https://github.com/SELinuxProject/selinux/blob/master/libsepol/src/policydb.c
+  https://github.com/SELinuxProject/selinux/blob/master/libsepol/src/write.c
+seq:
+  - id: header
+    type: header
+
+types:
+  header: # policydb_read
+    instances:
+      handle_unknown:
+        value: 'config & 6'
```

Comment:
What this instance is for? It's unused in the spec, and given that the bit mask `6` is `0b0110` in binary, I think it's quite unusual to use a bit mask that isn't shifted to the least significant bit and not do anything else with the result (I'd recognize a pattern `(x & 0b0110) >> 1`, or perhaps `x & 0b0110 != 0`, which is 99% of time used to test just a single bit though, but merely `x & 0b0110` is usually not that useful).

Reply:
It was weird that the reader is actually storing it even with no use of it, so I initially wanted to highlight it.